lib/tasks/one_time.rake

```diff
@@ -0,0 +1,8 @@
+namespace :one_time do
+  desc "Populate totp_seed from mfa_seed"
+  task populate_totp_seed: :environment do
+    User.find_each do |user|
+      user.update(totp_seed: user.mfa_seed)
```

Comment:
```suggestion
      user.update_column(totp_seed: user.mfa_seed)
```

`update_column` will directly update the database column to ensure the two columns are the same. `update` will need to go through validations + less efficient.

Reply:
Wouldn't be `User.where.not(mfa_seed: nil).update_all('totp_seed = mfa_seed')` enough? In this case I think it is ok to keep it in migration, since it is one-liner and rake task would be needed to be removed soon since mfa_seed method will get unavailable after migration is finished. But both works for mw.